src/routes/webhooks/paddle.ts

```diff
@@ -122,18 +99,6 @@ export const updateUserSubscription = async ({
         cycle: state ? subscriptionType : null,
         createdAt: state ? data.data?.startedAt : null,
         subscriptionId: state ? data.data?.id : null,
-        ...(isGift && {
```

Comment:
Since code was moved from here should we ignore gifter in `SubscriptionCreated` or is it not sent at all for OTP?

Reply:
🤝